src/main/java/com/matt/forgehax/mods/TimerMod.java

```diff
@@ -28,6 +32,15 @@ public TimerMod() {
           })
           .build();
 
+  public final Setting<Boolean> tpsSync =
+          getCommandStub()
+                  .builders()
+                  .<Boolean>newSettingBuilder()
+                  .name("tpsSync")
```

Comment:
Small suggestion - use the `name("tps-sync")`. Other than that everything looks good